plugins/tokenizers/mecab.c

```diff
@@ -1063,13 +1064,20 @@ mecab_next_default_format(grn_ctx *ctx,
   }
 }
 
+static int
+mecab_wakati_delimiter(const char *str)
```

Comment:
If we don't use a space for delimiter, it's not called "wakati gaki". So we should not use "wakati" in our code.

Reply:
Absolutely.

Reply:
fix: [5c9f82b](https://github.com/groonga/groonga/pull/2331/commits/5c9f82be569bb548b3360f3e4709e8f56e321850) I renamed it from `'wakati'` to `'newline'`.